docs/layers/graph-layer.md

```diff
@@ -0,0 +1,35 @@
+<!-- INJECT:"GraphLayerDemo" -->
```

Comment:
Since everything else is in the `/examples` folder (a good initial approach I think), this doc would be better as a README.md in the layer folder in the example at this point rather than part of the official layer documentation.